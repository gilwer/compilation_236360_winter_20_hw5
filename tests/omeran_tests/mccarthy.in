int mccarthy(int n){
	if (n > 100){
		return n - 10;
	}
	else {
		return mccarthy(mccarthy(n + 11));
	}
}

void main(){
	if (mccarthy(120) == 110){
		print("Mccarthy was right!");
	}
	else {
		print("Mccarthy was wrong...");
	}

	if (mccarthy(50) == 91){
		print("Mccarthy was right again");
	}
	print("Yea..");
}
