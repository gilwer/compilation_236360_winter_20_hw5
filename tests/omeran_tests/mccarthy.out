Mccarthy was right!
Mccarthy was right again
Yea..
